### DISCUSSION TITLE
What are the hyper parameter in SAD?

### DISCUSSION THREAD

**Discussion by rohansharma777** [pyannote/pyannote-audio#772] [Q&A] [answered]
I want to know the meaning of this parameters or where can I find details about it.

`binarize = Binarize(offset=0.8, onset=0.8, log_scale=True,
                        min_duration_off=0.1, min_duration_on=0.1)`

Specifically what are the offset, onset parameters.

**Comment by hbredin** [comment, marked as answer]
See Figure 2 of [this paper](https://arxiv.org/abs/2104.04045).
<img width="638" alt="image" src="https://user-images.githubusercontent.com/1080837/135528078-805b94fa-b92a-4baa-adf0-dfd585e3df6b.png">

**Reply by rohansharma777** [reply]
Thanks for the info @hbredin . The pyannote-audio library is awesome.
